docs/api-reference/next/link.md

```diff
@@ -54,7 +54,7 @@ export default Home
 
 `Link` accepts the following props:
 
-- `href` - The path or URL to navigate to. This is the only required prop
+- `href` - The path or URL to navigate to. This is the only required prop. It can also be an object, see below  
```

Comment:
```suggestion
- `href` - The path or URL to navigate to. This is the only required prop. It can also be an object, see [example here](/docs/api-reference/next/link.md#with-url-object)  
```